.github/workflows/submitty_ci.yml

```diff
@@ -600,6 +520,76 @@ jobs:
           - name: Check for site errors
             working-directory: ${{ env.SUBMITTY_REPOSITORY }}
             run: sudo bash tests/test_site_error_log.sh
+    e2eIntegrationTests:
+        needs:
+          - js-lint
+          - js-unit
+          - php-lint
+          - php-unit
+          - python-lint
+          - python-unit
+          - css-lint
+          - shellcheck
+        runs-on: ubuntu-20.04
+        services:
+          postgres:
+            image: postgres
+            env:
+              POSTGRES_PASSWORD: submitty_dbuser
+              POSTGRES_USER: postgres
+              TZ: America/New_York
+            options: >-
+              --health-cmd pg_isready
+              --health-interval 10s
+              --health-timeout 5s
+              --health-retries 5
+            ports:
+              - 5432:5432
+
+        steps:
```

Comment:
I think most of the stuff in this section can be eliminated.  The integration tests are mostly unit tests, and don't need the sample courses or website to be working as far as I am aware.